src/components/Inventory/withdraw/BatchWithdrawFlow.tsx

```diff
@@ -32,13 +75,13 @@ export default function BatchWithdrawFlow(props: BatchWithdrawFlowProps): JSX.El
 
   return (
     <>
-      {flowState === 'purpose' && (
+      {flowState === 'purpose' && batches && (
```

Comment:
i think we can move the batches check above?

```
if (!batches) {
  return <CircularProgress />
}
```
so we can avoid the batches check on all the components below

Reply:
Good idea